system-test/test-e2e.ts

```diff
@@ -43,7 +43,7 @@ interface Child {
   process?: cp.ChildProcess;
 }
 
-// This test could take up to 70 seconds.
+// This test could take up to 100 seconds.
```

Comment:
Was this failing on machine with 70 seconds because of a timeout, but was succeeding with 100 seconds?

Reply:
The test always succeeds without timing out, but it ran for about 100 seconds, so I thought the comment was misleading. Do you think it should be removed entirely?

Reply:
In that case, I think the comment is good.